### PR TITLE
Change code to return c2c request which the same to quickcall

### PR DESCRIPTION
**Pull request by emnvn**
This change work for https://2600hz.atlassian.net/browse/KAZOO-4099

### CODE REVIEW

File: deps/gen_smtp-8f7ba01172e679e8b4262334cefc3451b4c36777/testdata/dkim-rsa-private-encrypted:12345.pem [hunk 1]
```diff
@@ -1,18 +0,0 @@
------BEGIN RSA PRIVATE KEY-----
```
Comment (jamesaimonetti) [inline deps/gen_smtp-8f7ba01172e679e8b4262334cefc3451b4c36777/testdata/dkim-rsa-private-encrypted:12345.pem]: Why is this deleted file in the PR?

File: applications/crossbar/src/modules/cb_clicktocall.erl [hunk 2]
```diff
@@ -447,6 +449,7 @@ build_originate_req(Contact, Context) ->
        ,{<<"Outbound-Callee-ID-Number">>, CalleeNumber}
        ,{<<"Outbound-Caller-ID-Name">>, FriendlyName}
        ,{<<"Outbound-Caller-ID-Number">>, OutboundNumber}
+	   ,{<<"Outbound-Call-ID">>, CallId}
```
Comment (k-anderson) [inline applications/crossbar/src/modules/cb_clicktocall.erl]: @emnvn 
Howdy :)

Setting the Outbound-Call-ID on the root of the originate request will cause it to fail if there are multiple endpoints (they will all attempt to use this call-id).  Instead a unique call id should be generate and set (same property name) on each of the 'Endpoints'.

Comment (fenollp) [inline applications/crossbar/src/modules/cb_clicktocall.erl]: @emnvn As @k-anderson says, have a look at cb_modules_util in #1233. It sets a per-endpoint outbound call id

**Comment by k-anderson** [comment]
@emnvn 
Anything we can do to help with this?

**Comment by fenollp** [comment]
@emnvn CI is a bit old; please consider rebasing.
Also, build fails because there are trailing spaces in `applications/crossbar/src/modules/cb_clicktocall.erl`

File: deps/gen_smtp-8f7ba01172e679e8b4262334cefc3451b4c36777/testdata/dkim-rsa-private-encrypted:12345.pem (hunk 1 above)
Comment (emnvn) [inline deps/gen_smtp-8f7ba01172e679e8b4262334cefc3451b4c36777/testdata/dkim-rsa-private-encrypted:12345.pem]: Oh, I did not delete this file, but I don't know why it was deleted when I create PR.

**Comment by jamesaimonetti** [comment]
@emnvn one more note; please respect formatting (use spaces, not tabs), so that things line up properly.

**Comment by lazedo** [comment]
@emnvn please rebase and address the formatting comment.

File: applications/crossbar/src/modules/cb_clicktocall.erl (hunk 2 above)
Comment (lazedo) [inline applications/crossbar/src/modules/cb_clicktocall.erl]: @fenollp @k-anderson looking at https://github.com/2600hz/kazoo/pull/1233 it seems that it sets the outbound-call-id per endpoint but with the same call-id for all endpoints

**Comment by lazedo** [comment]
closed in favor of https://github.com/2600hz/kazoo/pull/1416
